### DISCUSSION TITLE
Vertex Color Error

### DISCUSSION THREAD

**Discussion by watercaustics** [BramStoutProductions/MiEx#87] [Q&A]
When exporting a very large 3k x 3k file into USD format after all MiEx files were generated, I get this error:

Could not write node MATfor texture vertexcolor:vertexcolor
java.lang.NullPointerException: Cannot invoke "String.contains(java.lang.CharSequence)" because "node.type" is null

**Comment by BramStout** [comment]
It looks like a small bug with the material templates not supporting exported maps in item frames. I've updated them, so they should now support maps in item frames. Here's a zip with updated templates for the base_resource_pack and the UsdPreviewSurface resource packs. Please try extracting this into the `./resources` folder next to the `MiEx.jar` file, to replace the existing material template files in there. If you used any of the example resource packs found in the repository, I've also updated them, so you can simply redownload and replace them.

Please let me know if this fixed your issue!

[fixed_material_templates.zip](https://github.com/user-attachments/files/15882565/fixed_material_templates.zip)
